openmp/runtime/src/kmp_io.cpp

```diff
@@ -50,6 +50,7 @@ static HANDLE __kmp_stderr = NULL;
 static int __kmp_console_exists = FALSE;
 static kmp_str_buf_t __kmp_console_buf;
 
+#if 0
 static int is_console(void) {
```

Comment:
Should we remove this function? It's there since the project was imported (5e8470af093f8d8106ca22e37133b41e0bdc5e85) but it has never been used.

Reply:
Sounds good to me.

Reply:
Removed.